<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtMUCSearchWindow</class>
 <widget class="QWidget" name="QtMUCSearchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>526</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Find Room</string>
  </property>
  <widget class="QTreeView" name="results_">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>256</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Available Rooms</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="service_">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>201</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Search another service:</string>
   </property>
  </widget>
  <widget class="QLabel" name="filterLabel_">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Only show rooms matching:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filter_">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Join Room Directly:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Your nickname:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nickName_">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="room_">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>400</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="joinButton_">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>480</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Join Room</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="showEmptyRooms_">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Empty Rooms</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton_">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>170</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="joinAutomatically_">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>440</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Join automatically in future</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
